ingest/src/settings.rs

```diff
@@ -0,0 +1,78 @@
+use crate::{error::DecodeError, Error, Result};
+use config::{Config, Environment, File};
+use serde::Deserialize;
+use std::{net::SocketAddr, path::Path, str::FromStr};
+
+#[derive(Debug, Deserialize)]
+pub struct Settings {
+    /// RUST_LOG compatible settings string. Default
+    /// "poc_ingest=debug,poc_store=info"
+    #[serde(default = "default_log")]
+    pub log: String,
+    /// Mode to run the server in (lora or mobile). Reuquired
+    pub mode: Mode,
+    /// Listen address. Required. Default is 0.0.0.0:9081
```

Comment:
listing defaults inline here means 3 places they must be updated, should we ever need to, maybe they're not needed here?

Reply:
Good point. Default-getting function should be sufficiently helpful for code reading.